src/styles/Form.module.css

```diff
@@ -0,0 +1,68 @@
+
+.formmaincontainer {
+  display: flex;
+  justify-content: center;
+  background-color: #e8e8dc;
+  width: 100%;
+}
+
+.formcontainer {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin-bottom: 200px;
+}
+
+/* line */
+.line2 {
+  height: 0.125rem;
+  background: #f5f6fa;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.form {
+  display: flex;
+  justify-content: center;
+  gap: 20px;
+}
+
+.titleinput {
+  width: 40%;
+  height: 1vh;
+  padding: 0.813rem 20rem 0.813rem 1.063rem;
+  border: 2px solid#e8e8e8;
+  border-radius: 4px;
+}
+
+.authorinput {
+  width: 30%;
+  height: 1vh;
+  padding: 0.813rem 5rem 0.813rem 1.063rem;
+  border: 2px solid#e8e8e8;
+  border-radius: 4px;
+}
+
+.categoryinput {
+  width: 50%;
+  height: 6vh;
+  padding: 0.375rem 0.938rem;
+  border: 2px solid#e8e8e8;
+  border-radius: 4px;
+}
+
+.inputform {
+  width: 28rem;
+  height: 1.625rem;
+  margin-top: 2rem;
+  background-color: white;
+}
+
+.submitbtn {
+  width: 10%;
+  height: 6vh;
+  background-color: #0290ff;
+  color: #fff;
+  border: 2px solid#e8e8e8;
+  border-radius: 4px;
```

Comment:
Please fix the height of the categories and submit button. They need to have the same width with the other input areas.
<img width="629" alt="Screenshot 2023-03-18 at 2 01 43 PM" src="https://user-images.githubusercontent.com/25887253/226102146-6e387c91-24db-416a-920d-b31fc2b45f7e.png">

<img width="1099" alt="Screenshot 2023-03-18 at 2 01 32 PM" src="https://user-images.githubusercontent.com/25887253/226102145-c816d6b6-e15c-4eb4-919e-c7930d57bdd2.png">